### PR TITLE
fix installMoby calls in vhdBuilder

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

File: pkg/agent/baker_test.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 		// customData
 		customData := baker.GetNodeBootstrappingPayload(config)
 		// Uncomment below line to generate test data in local if agentbaker is changed in generating customData
-		// backfillCustomData(folder, customData)
+		backfillCustomData(folder, customData)
```
Comment (yizhang4321) [inline pkg/agent/baker_test.go:123]: You need to re-comment out the code that writes test data?

File: pkg/agent/baker_test.go [hunk 2]
```diff
@@ -130,7 +130,7 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 		// CSE
 		cseCommand := baker.GetNodeBootstrappingCmd(config)
 		// Uncomment below line to generate test data in local if agentbaker is changed in generating customData
-		// ioutil.WriteFile(fmt.Sprintf("./testdata/%s/CSECommand", folder), []byte(cseCommand), 0644)
+		ioutil.WriteFile(fmt.Sprintf("./testdata/%s/CSECommand", folder), []byte(cseCommand), 0644)
```
Comment (yizhang4321) [inline pkg/agent/baker_test.go:133]: Same here. You need to re-comment out the code that writes test data?

Comment (eriksywu) [reply pkg/agent/baker_test.go]: oops thanks for catching

**Review by yizhang4321 — APPROVED** [review]

**Review by bowang-666 — APPROVED** [review]

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 3]
```diff
@@ -126,7 +126,7 @@ installContainerd() {
     CURRENT_VERSION=$(containerd -version | cut -d " " -f 3 | sed 's|v||' | cut -d "+" -f 1)
     # v1.3.7 is our lowest supported version of containerd
     local BASE_VERSION="1.3.7"
-    if semverCompare ${CURRENT_VERSION} ${BASE_VERSION}; then
+    if semverCompare ${CURRENT_VERSION:-"0.0.0"} ${BASE_VERSION}; then
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/cse_install.sh:129]: this needs to be cherry pick to branch official/v20200909

Comment (xuto2) [reply parts/linux/cloud-init/artifacts/cse_install.sh:129]: https://github.com/Azure/AgentBaker/pull/323

**Pull request merged**